components/nimbus/src/tests/stateful/test_behavior.rs

```diff
@@ -1443,4 +1445,80 @@ mod event_store_tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_integration_smoke_test() -> Result<()> {
```

Comment:
This test name could be a little more descriptive. What is actually being tested?